nasa/harmony
# Harmony 1375 demo notebook

## Jira Issue ID
HARMONY-1375

## Description
Changes the 'Demo Notebook' link from a link to static HTML to the github page for the notebook, which github will render. Also adds a 'Open in CoLab' badge at the top of the notebook that the user can click on in the rendered github page. This will open the notebook in Google Colab and the user can run the notebook there.

## Local Test Steps
1. Check out branch harmony-1375-demo-notebook
2. Run `npm start-dev-fast` to start Harmony
3. Go to http://localhost:3000 to get the landing page
4. Click on the 'View Jupyter Notebook Demo' link to open the notebook blob page in github
5. In the browser change the link so that 'main' is 'harmony-1375-demo-notebook' to load the blob from this branch.
6. Verify that the notebook renders without results and has an 'Open in Colab' badge at the otp.
7. Click on the badge
8. Verify that the notebook opens in Google Colab (you may need to create a free Google account if you don't have one)
9. Change the link to the notebook - chance 'main' to 'harmony-1375-demo-notebook' once again to Colab loads the notebook from github branch 'harmony-1375-demo-notebook'
10. Run the notebook (you will need to enter your UAT EDL credentials)
NOTE: some of the WMS cells are broken due to a Harmony regression - these should start working when HARMONY-1382 is worked. They are broken in the 'main' branch as well.

## PR Acceptance Checklist
